packages/card/src/Card.ts

```diff
@@ -263,7 +258,7 @@ export class Card extends LikeAnchor(
 
     private get renderSubtitleAndDescription(): TemplateResult {
         return html`
-            <div class="subtitle spectrum-Detail spectrum-Detail--sizeS">
+            <div class="subtitle">
```

Comment:
this is where im seeing the issue in VRT

Reply:
@castastrophe looooks so much closer, seems like letter-spacing is the only thing off in the VRTs but I personally like the actual over the baseline. is this an acceptable difference?

Reply:
Yeah I kind of like the updated letter spacing for readability.